app/src/main/java/org/simple/clinic/bp/entry/BloodPressureEntrySheetController.kt

```diff
@@ -162,7 +162,7 @@ class BloodPressureEntrySheetController @Inject constructor(
 
     val dateForNewBp = openAsStream
         .ofType<New>()
-        .map { LocalDate.now(userClock) }
+        .map { LocalDate.now() }
```

Comment:
Why did we remove the `userClock` parameter when constructing this instance?

Reply:
Can't we just remove the `dateForNewBp` stream instead?

Reply:
Fixed this by putting back the user clock.

 We can't remove the `dateForNewBp` stream now because save entry is dependent on the state of this stream. We can remove this only after we migrate the save entry functionality.